conf/vertebrates/mlss_conf.xml

```diff
@@ -125,11 +126,22 @@
     </multiple_alignment>
 
     <!-- Pigs -->
-    <multiple_alignment method="EPO+EPO_LOW_COVERAGE" gerp="1">
-      <species_set name="sus">
-        <taxonomic_group taxon_name="Sus"/>
+    <!-- Special case: base EPO should remain internal/unreleased, but should still be updated -->
+    <multiple_alignment method="EPO" no_release="1">
+      <species_set name="livestock" display_name="livestock">
+        <genome name="sus_scrofa"/>
+        <genome name="sus_scrofa_usmarc"/>
+        <genome name="bos_taurus"/>
+        <genome name="ovis_aries"/>
+        <genome name="equus_caballus"/>
+      </species_set>
+    </multiple_alignment>
+    <multiple_alignment method="EPO_LOW_COVERAGE" gerp="1">
+      <species_set name="pig_breeds" display_name="pig breeds" in_collection="livestock">
+        <taxonomic_group taxon_name="Eutheria"/> <!-- everything in the collection -->
```

Comment:
It may work without this `<taxonomic_group>` line

Reply:
Yes, just tested it - it does work without. Pushing new commit now.

Reply:
I've just tested it and it seems to require the `<taxonomic_group>`